src/content/docs/dev/amplifier/chain-integration/integrate-a-chain.mdx

```diff
@@ -94,9 +94,13 @@ You can either use the existing deployed contracts and on the devnet via their `
 <tab-item title="Use pre-deployed contracts">
 Use the existing deployments for all three contracts, making note of the existing deployment's `code_id`:
 
-- Voting verifier (code_id: `626`)
```

Comment:
Does that mean this code ID is no longer working? Isn't this a permanent contract deployment? What changed?